sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/MetricCell.java

```diff
@@ -28,7 +29,17 @@
  * @param <DataT> The type of metric data stored (and extracted) from this cell.
  */
 @Experimental(Kind.METRICS)
-public interface MetricCell<UserT extends Metric, DataT> {
+public interface MetricCell<UserT extends Metric, DataT> extends Serializable {
+
+  /**
+   * Update value of this cell.
+   */
+  void update(DataT data);
```

Comment:
It does a set, which is why I chose `update` over `merge`, void return type and stated in the Javadoc that it updates the value of the cell.
Is this unclear? Should I use a different method name?